### DISCUSSION TITLE
upload big file via browser error, file > 10GB

### DISCUSSION THREAD

**Discussion by ccKris** [minio/minio#16582] [Q&A] [answered]
Hello, I am trying to upload a big file to my bucket, but I was not able to do so via Browser。
the file size > 10GB, it give me "Error: Error - A network error occurred".
minio version is RELEASE.2023-01-31T02-24-19Z，on windows

but the older minio version ( RELEASE.2020-11-10T21-02-24Z) support that.
 
Thanks in advance for the help.

**Comment by harshavardhana** [comment, marked as answer]
Please report this issue to https://github.com/minio/console/issues - this mostly looks like your setup issue.

**Reply by klauspost** [reply]
If I recall correctly Console uploads can only be up to 5GB.
